operate/client/src/modules/stores/autoMapping.ts

```diff
@@ -1,73 +0,0 @@
-/*
```

Comment:
Renamed to processInstanceMigrationMapping.ts, because it now contains general mapping functionality (not only auto-mapping)

To see the diff, please check [this commit](https://github.com/camunda/camunda/pull/22179/commits/eb05ea3c92d6828bdbf9c6f5af3bdda05b6cd667)